spec/lib/definition/model_spec.rb

```diff
@@ -5,7 +5,7 @@
 describe Definition::Model do
   let(:test_model_class) do
     Class.new(described_class) do
-      required :name, Definition.Type(String)
+      required :name, Definition.NonEmptyString
```

Comment:
```suggestion
required :name, Definition.Lambda { |value| conform_with(value.trim) }
```
to have a test case that would cause the error that this pr fixes